test/output/only-nested/expected.js

```diff
@@ -0,0 +1,3 @@
+"use strict";function f() {var a,c;module.importSync("./module",{"a":function(v){a=v},"b":function(v){c=v}},0);
+
+}
```

Comment:
This test case wouldn't work without the `visitProgram` method, because we wouldn't have a `bodyInfo` for the top level (only the `f` function scope).